pkg/dsync/drwmutex_test.go

```diff
@@ -88,12 +91,16 @@ func TestSimpleWriteLockTimedOut(t *testing.T) {
 	}
 }
 
+func TestCancelContextIfNotRefreshed(t *testing.T) {
+}
+
```

Comment:
were you planning to add a test here? @vadmeste

Reply:
> were you planning to add a test here? @vadmeste

No, that was a typo. I already added a test in dsync_test.go